pico/src/temp.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 Space Cubics, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/i2c.h>
+
+#define DEV_ADDR   (0x4e)
+#define START_ADDR (0x00)
+#define RESOLUTION (0.0625)
+#define BIT_SHIFT  (4)
+
+int get_temp(const struct device *dev, float *temp)
```

Comment:
Because it is now here, you didn't 'remove' it, but rather 'moved' it "from" `main.c` to `temp.c`. And yes, you removed its calling in `main()`, but you didn't explain why.